src/main/java/tech/jhipster/lite/generator/server/sonar/domain/Sonar.java

```diff
@@ -0,0 +1,17 @@
+package tech.jhipster.lite.generator.server.sonar.domain;
+
+public class Sonar {
+
+  public static final String SONARSOURCE_MAVEN_PLUGIN_VERSION = "3.9.1.2184";
+  public static final String SONARQUBE_DOCKER_VERSION = "9.2.4-community";
```

Comment:
maybe doing the same than in MySQL, by including the image name too?

```suggestion
  public static final String SONARQUBE_DOCKER_IMAGE = "sonarqube:9.2.4-community";
```